src/webgpu/capability_info.ts

```diff
@@ -817,6 +777,7 @@ const [kLimitInfoKeys, kLimitInfoDefaults, kLimitInfoData] =
   'maxComputeWorkgroupSizeY':                  [           ,       256,             128,                          ],
   'maxComputeWorkgroupSizeZ':                  [           ,        64,              64,                          ],
   'maxComputeWorkgroupsPerDimension':          [           ,     65535,           65535,                          ],
+  // DO NOT ADD OPTIONAL limits here for proposals to the spec. Any limit here is immediately required by all implementations.
```

Comment:
I guess this works because we don't have a static-assertion right now that this table has all the limits from the types? But we _should_ have that, so this table doesn't get outdated.